src/ui/src/partials/job_index.html

```diff
@@ -1,5 +1,22 @@
 <h1 class='page-header'>
   <span>Request Scheduler</span>
+  
+  <button type="button"
```

Comment:
I know you mentioned in your directions that this button only shows if there are already jobs, but that seems like a think worth fixing.  Presumably one of the use cases is someone standing up a new garden and they want to import some previously exported set of jobs.  It seems weird to make them create a job first just to get the import option to show up.

Reply:
Yes, this is right. I hadn't considered the use case of importing when there were yet no Jobs. Will fix.